grakn-test/src/test/java/ai/grakn/test/engine/GraknEngineServerIT.java

```diff
@@ -99,7 +100,7 @@ public void whenSendingTasksToTwoEngines_TheyAllComplete(
     @Property(trials=10)
     public void whenEngine1StopsATaskBeforeExecution_TheTaskIsStopped(@NewTask TaskState task) {
         String uri1 = "localhost:" + PORT1;
-        TaskClient.of(uri1).stopTask(task.getId());
+        assertTrue(TaskClient.of(uri1).stopTask(task.getId()));
```

Comment:
Just for the sake of getting more info form the logs it might be worth adding a line as to what the failure may be. e.g. `assertTrue("Failed to stop task, ["+ task.getId + "]", ...)`